<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AircraftWidget</class>
 <widget class="QWidget" name="AircraftWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="saveAircraftToSD">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>93</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Be sure to set the Neutral position on all sliders before sending!
Applies and Saves all settings to SD</string>
    </property>
    <property name="text">
     <string>Save to SD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveAircraftToRAM">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>93</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Send to OpenPilot but don't write in SD.
Be sure to set the Neutral position on all sliders before sending!</string>
    </property>
    <property name="text">
     <string>Save to RAM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getAircraftCurrent">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>93</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Retrieve settings from OpenPilot</string>
    </property>
    <property name="text">
     <string>Get Current</string>
    </property>
   </widget>
   <widget class="QComboBox" name="aircraftType">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Select aircraft type here</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Aircraft type:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
